fulfillment.go

```diff
@@ -0,0 +1,150 @@
+}
+
+// FulfillmentsResource represents the result from the fullfilments.json endpoint
+type FulfillmentsResource struct {
+	Fulfillments []Fulfillment `json:"fulfillments"`
+}
+
+// List fulfillments
+func (s *FulfillmentServiceOp) List(options interface{}) ([]Fulfillment, error) {
+	prefix := FulfillmentPathPrefix(s.resource, s.resourceID)
+	path := fmt.Sprintf("%s.json", prefix)
+	resource := new(FulfillmentsResource)
+	err := s.client.Get(path, resource, options)
+	return resource.Fulfillments, err
+}
+
+// Count fulfillments
+func (s *FulfillmentServiceOp) Count(options interface{}) (int, error) {
+	prefix := FulfillmentPathPrefix(s.resource, s.resourceID)
+	path := fmt.Sprintf("%s/count.json", prefix)
+	return s.client.Count(path, options)
+}
+
+// Get individual fulfillment
+func (s *FulfillmentServiceOp) Get(fulfillmentID int, options interface{}) (*Fulfillment, error) {
+	prefix := FulfillmentPathPrefix(s.resource, s.resourceID)
+	path := fmt.Sprintf("%s/%d.json", prefix, fulfillmentID)
+	resource := new(FulfillmentResource)
+	err := s.client.Get(path, resource, options)
+	return resource.Fulfillment, err
+}
+
+// Create a new fulfillment
+func (s *FulfillmentServiceOp) Create(fulfillment Fulfillment) (*Fulfillment, error) {
+	prefix := FulfillmentPathPrefix(s.resource, s.resourceID)
+	path := fmt.Sprintf("%s.json", prefix)
+	wrappedData := FulfillmentResource{Fulfillment: &fulfillment}
+	resource := new(FulfillmentResource)
+	err := s.client.Post(path, wrappedData, resource)
+	return resource.Fulfillment, err
+}
+
+// Update an existing fulfillment
+func (s *FulfillmentServiceOp) Update(fulfillment Fulfillment) (*Fulfillment, error) {
+	prefix := FulfillmentPathPrefix(s.resource, s.resourceID)
+	path := fmt.Sprintf("%s/%d.json", prefix, fulfillment.ID)
+	wrappedData := FulfillmentResource{Fulfillment: &fulfillment}
+	resource := new(FulfillmentResource)
+	err := s.client.Put(path, wrappedData, resource)
+	return resource.Fulfillment, err
+}
+
+// Complete an existing fulfillment
+func (s *FulfillmentServiceOp) Complete(fulfillmentID int) (*Fulfillment, error) {
+	prefix := FulfillmentPathPrefix(s.resource, s.resourceID)
+	path := fmt.Sprintf("%s/%d.json", prefix, fulfillmentID)
+	resource := new(FulfillmentResource)
+	err := s.client.Post(path, nil, resource)
+	return resource.Fulfillment, err
+}
+
+// Transition an existing fulfillment
+func (s *FulfillmentServiceOp) Transition(fulfillmentID int) (*Fulfillment, error) {
+	prefix := FulfillmentPathPrefix(s.resource, s.resourceID)
+	path := fmt.Sprintf("%s/%d.json", prefix, fulfillmentID)
+	resource := new(FulfillmentResource)
+	err := s.client.Post(path, nil, resource)
+	return resource.Fulfillment, err
+}
+
+// Cancel an existing fulfillment
+func (s *FulfillmentServiceOp) Cancel(fulfillmentID int) (*Fulfillment, error) {
+	prefix := FulfillmentPathPrefix(s.resource, s.resourceID)
+	path := fmt.Sprintf("%s/%d.json", prefix, fulfillmentID)
```

Comment:
I think this needs to be:
`path := fmt.Sprintf("%s/%d/cancel.json", prefix, fulfillmentID)`